src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformTabularTests.cs

```diff
@@ -278,8 +272,9 @@ public async Task SP_Tabular()
 #pragma warning restore CS0618 // Type or member is obsolete
 
             SymbolValues symbolValues = new SymbolValues().Add("Documents", spTable);
-            BaseRuntimeConnectorContext runtimeContext = new TestConnectorRuntimeContext(tabularService.Namespace, client, console: _output);
-            RuntimeConfig rc = new RuntimeConfig(symbolValues).AddRuntimeContext(runtimeContext);
+            RuntimeConfig rc = new RuntimeConfig(symbolValues)
+                                    .AddService<ConnectorLogger>(logger)
+                                    .AddService<HttpClient>(client);
```

Comment:
>       .AddService<HttpClient>(client); [](http://example.com/codeflow?start=29&length=39)

This will be a problem.  If we have 2 tables, they can compete over which httpClient this maps to.
HttpClient is too general of a type to use here.   This was what BaseRuntimeConnectorContext was trying to solve. 

Alternatively, I think the TableValue can just hold onto the client directly. (This is what Dataverse does)

Reply:
Filed #2408 to followup